internal/ingress/controller/store/store_test.go

```diff
@@ -740,3 +740,76 @@ func TestUpdateSecretIngressMap(t *testing.T) {
 		}
 	})
 }
+
+// Test to avoid regression fixed in https://github.com/kubernetes/ingress-nginx/pull/2361
+func TestListIngressWithoutPath(t *testing.T) {
```

Comment:
From what I understand you're actually testing the function completely rather than one aspect of it, therefore I think this should just be called `TestListIngresses`.

IMHO the comment does not add any value.

Reply:
done